components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/util/OAuth2Util.java

```diff
@@ -166,6 +166,11 @@
     public static final String OPENID_CONNECT = "OpenIDConnect";
     public static final String ENABLE_OPENID_CONNECT_AUDIENCES = "EnableAudiences";
     public static final String OPENID_CONNECT_AUDIENCE = "audience";
+    /*
+     * Maintain a separate parameter "OPENID_CONNECT_IDENTITY_CONFIG_AUDIENCE" for get the audience form the identity.xml
+     * when user didn't add any audience in the dashboard while creating service provider.
```

Comment:
Do not say dashboard, as we already have a seprate dashboad application IS.
Say management console or UI